common/models/user.js

```diff
@@ -760,8 +760,7 @@ module.exports = function(User) {
 
             return tokenID;
           }, description: 'Do not supply this argument, it is automatically extracted ' +
-            'from request headers.',
-          },
+            'from request headers.'},
```

Comment:
my linter did the fix automatically according to loopback .jsrc, but ok, reverted